src/bone.js

```diff
@@ -4,7 +4,7 @@
 // pins is an object whose keys are the pinIndex keys, e.g. P9_14.
 // uarts and i2c are objects describing the serial ports and i2c buses.
 
-
+console.log("Loading bone.js");
```

Comment:
I think this debug statement can be removed.

Reply:
Done